dragon_gigapet/Dragon.rb

```diff
@@ -0,0 +1,39 @@
+class Dragon < Animal  # Do I still need to add instant variables to initalize if already in animal file 
+    
+
+
+public
+
+    def mood_check #Jonah mentioned that I shouldn't have puts statements in my class files. Only in 1st level files AKA Gigapet.rb
+        if @hunger >= 60
+            puts "The dragon seems to try to bite at your leg. They must be hungry?"
+        elsif @bored >= 60
+            puts "The dragon tries to jump on you? They must want to play. "
+        elsif @tired >= 60 
+            puts "The dragon attempts to sleep on the ground even though they just hatched. They must be bored?"
+        elsif @need_affection >= 60 
+            puts "The Dragon is rubbing you leg, seems to want somekind of affection?"
+        else
+            puts "Looks like the dragon is ready to leave!!"
+        end # is there a better loop to use instead of if, else statement. Maybe case statement 
+    end 
+
+
+
+# When you are able to figure out a way to reference toward the instance variables created in Animal.rb, you will need to change these methods 
+    def feed
+        @hunger - 100
```

Comment:
Is this new value meant to replace the old value? If so, we should assign the new value:
```suggestion
        @hunger = @hunger - 100
        # a shorter way to do this would be:
        # @hunger -= 100
```

Reply:
Good catch! If `@hunger` was 150, then calling `feed` would return 50. But `@hunger` would still have the value of 150. Did you want it to change after calling `feed`?

Reply:
This makes sense. Big picture, this will cause a lot of issues if I wanted to expand the project. Changed to @hunger -= 100.